### PR TITLE
[DPE-2375] Improve structure and functionalities of Github Central Uploader

### PR DESCRIPTION
**Pull request by welpaolo**

### CODE REVIEW

File: products.json [hunk 1]
```diff
@@ -2,29 +2,15 @@
...
         "artifactory-url": "ARTIFACTORY_DEPLOY_URL",
         "artifactory-user": "ARTIFACTORY_USER",
         "artifactory-token": "ARTIFACTORY_TOKEN",
         "tarball-regex": "spark-*.tgz"
-    },
-    {
-        "name": "opensearch",
-        "lp-building-project": "opensearch-lp-build",
-        "lp-releasing-project": "opensearch-releases",
-        "lp-building-repo": "~medib/opensearch-lp-build/+git/opensearch-lp-build",
```
Comment (Mehdi-Bendriss) [inline products.json:20]: just adding this for reference `lp-building-repo: ~data-platform/soss/+source/opensearch-build/+git/opensearch-build`

File: .github/workflows/release-tarball.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+name: Release CI
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
```
Comment (Mehdi-Bendriss) [inline .github/workflows/release-tarball.yaml:8]: nit: once the whole pipeline is stabilized we'll probably need to remove this

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    poetry run black --check --diff {[vars]all_path}
+
+[testenv:unit]
+description = Run unit tests
+commands =
+    poetry install --with unit --sync --no-cache
+    poetry export -f requirements.txt -o requirements.txt
+    poetry run pytest tests/unittest
+
+[testenv:integration]
```
Comment (deusebio) [inline tox.ini]: Since we don't have integration tests, I would remove this

File: tox.ini [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+description = Run integration tests
+setenv =
+    IE_TEST=1
+commands =
+    poetry install --with integration --sync --no-cache
+    poetry export -f requirements.txt -o requirements.txt
+    poetry run pytest tests/integration
+
+[testenv:all-tests]
```
Comment (deusebio) [inline tox.ini]: Let's remove this one as well, since we already have the unit one

**Review by deusebio — APPROVED** [review]

File: products.json [hunk 5]
```diff
@@ -2,29 +2,15 @@
...
         "lp-access-token": "LP_ACCESS_TOKEN_SPARK",
         "lp-access-secret": "LP_SECRET_TOKEN_SPARK",
         "artifactory-url": "ARTIFACTORY_DEPLOY_URL",
         "artifactory-user": "ARTIFACTORY_USER",
         "artifactory-token": "ARTIFACTORY_TOKEN",
         "tarball-regex": "spark-*.tgz"
-    },
-    {
-        "name": "opensearch",
-        "lp-building-project": "opensearch-lp-build",
```
Comment (Mehdi-Bendriss) [inline products.json:18]: just adding this for reference `lp-building-project: "opensearch-build"`

**Review by Mehdi-Bendriss — APPROVED** [review]

File: .github/workflows/release-tarball.yaml (hunk 2 above)
Comment (welpaolo) [reply .github/workflows/release-tarball.yaml:8]: Sure!

**Pull request merged**